modules/web/src/de/diedavids/cuba/dataimport/web/importwizard/ImportWizard.groovy

```diff
@@ -99,17 +104,46 @@ class ImportWizard extends AbstractWindow {
     @Inject
     ImportAttributeMapperCreator importAttributeMapperCreator
 
+    ImportConfiguration defaultImportConfiguration
+
+    @Inject
+    Button toStep3
+
     @Override
     void init(Map<String, Object> params) {
-
         entityLookup.setOptionsMap(entityClassSelector.entitiesLookupFieldOptions)
-
         importConfigurationDs.setItem(createImportConfiguration())
-
         initEntityClassPropertyChangeListener()
         initReusePropertyChangeListener()
         initImportFileHandler()
         initImportFileParser()
+        initTitles()
+        initConfigLookupChange()
+    }
+
+    void initConfigLookupChange() {
+        configLookup.addValueChangeListener(new Component.ValueChangeListener() {
```

Comment:
can be an explicit class `ConfigLookupValueChangeListener` to reduce the class with Class attributes like in this example:
https://github.com/mariodavid/cuba-component-user-inbox/blob/master/modules/web/src/de/diedavids/cuba/userinbox/web/message/ToggleReadActionListener.groovy